website/src/components/Performance.astro

```diff
@@ -0,0 +1,174 @@
+---
+import Inputf from "@src/components/formatter/input.md";
+import Outputf from "@src/components/formatter/output.md";
+import ProgressBar from "../../src/playground/components/Progress.tsx";
+---
+
+<div class="hero-code">
+  <div class="formatter">
+    <div class="head">Biome is fast</div>
```

Comment:
Would be good to use <h1> here instead of div, so you don't need to style it because it is styled by Starlight

Reply:
I am not sure we should have an `h1`. Having multiple `h1` will penalise the SEO. The home page already has one (the hero). Maybe an `h2`?

Reply:
You are right, `h2` is better in that case

Reply:
We can still assign a custom class and increase the font size. However, keeping starlight styles is the priority